app/connectors/TtpConnector.scala

```diff
@@ -22,7 +22,7 @@ import play.api.mvc.RequestHeader
 import requests.RequestSupport._
 import uk.gov.hmrc.http.HttpClient
 import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
-import uk.gov.hmrc.http.HttpReads.Implicits
+import uk.gov.hmrc.http.HttpReads.Implicits._
```

Comment:
Please mind the `HttpReadsUnitThrowingException` when importing that wildcard.

https://github.com/hmrc/essttp-backend/blob/main/cor-journey/src/main/scala/essttp/utils/HttpReadsUnitThrowingException.scala

Reply:
:+1:  I've made the import more precise [here](https://github.com/hmrc/essttp-frontend/pull/46/commits/f21a1447f5c73e9ae1ba9ab708c055f6ccb285d7) now

Reply:
to future proof I've just made sure the default `readUnit` is not brought in [here](https://github.com/hmrc/essttp-frontend/pull/46/commits/3eb80cd751658a9af7193e75f0e28e6b49ef0ef0)